src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/Summary.tsx

```diff
@@ -61,6 +61,30 @@ export function Summary({
             {t(PurchaseOverviewTexts.summary.messageInZone)}
           </ThemeText>
         );
+      case 'multiple-stop-harbor':
+        const harbortext = (
+          <ThemeText type="body__secondary" style={styles.message}>
+            {t(PurchaseOverviewTexts.summary.messageInHarborZones)}
+          </ThemeText>
+        );
+        if (fareProductTypeConfig.type === 'boat-period')
+          return (
+            <>
+              <ThemeText type="body__secondary" style={styles.message}>
```

Comment:
`<ThemeText type="body__secondary" style={styles.message}>< /ThemeText >` is repeated a lot in this switch. Could consider something like
`const SummaryThemeText = (text: string) => (
      <ThemeText type="body__secondary" style={styles.message}>
        {text}
      </ThemeText>
    );`

And then just `SummaryThemeText(t(PurchaseOverviewTexts.summary.messageInZone))`